### PR TITLE
edit_rstudio_snippets path

### PR DESCRIPTION
**Pull request by cstepper**
Hi, 
starting from RStudio Version 1.3, the default directories for storing the *config* files has changed (https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/).
I adjusted the path construction part of `edit_rstudio_snippets()` to account for this - taking into account the 
- RStudio Version
- OS type
Not sure if the way I created the conditional statement is the most elegant - please feed free to edit!
Thanks, Christoph

### CODE REVIEW

File: R/edit.R [hunk 1]
```diff
@@ -103,7 +104,20 @@ edit_rstudio_snippets <- function(type = c("r", "markdown", "c_cpp", "css",
...
+  # 1) RStudio Version
+  # 2) OS.type
+  # https://blog.rstudio.com/2020/02/18/rstudio-1-3-preview-configuration/
+  if (rstudioapi::versionInfo()$version < "1.3") {
+    path <- path_home_r(".R", "snippets", path_ext_set(type, "snippets"))
+  } else {
+    if (.Platform$OS.type == "windows") {
+      path <- path_home_r("AppData/Roaming/Rstudio", "snippets", path_ext_set(type, "snippets"))
+    } else {
+      path <- path_home_r(".config/rstudio", "snippets", path_ext_set(type, "snippets"))
```
Comment (jennybc) [inline R/edit.R:117]: I feel a little queasy hard-wiring this here. Does rstudioapi offer a way to get (more of) these paths? I'd rather consult an outside authority.

Comment (cstepper) [reply R/edit.R:117]: Yes, this sounds reasonble. I couldn't find an `rstudioapi` function here. But maybe you can consult someone of the develepers!

Comment (jennybc) [reply R/edit.R:117]: https://github.com/rstudio/rstudioapi/issues/176

**Comment by jennybc** [comment]
I've re-pinged the IDE team re: advice on these path matters.

**Comment by hadley** [comment]
Superseded by #1278 — thanks for starting on this! I ended up using rappdirs to get the full treatment of the XDG spec.

**Pull request closed without merging**